plugins/amazonq/chat/jetbrains-community/src/software/aws/toolkits/jetbrains/services/amazonqFeatureDev/session/CodeGenerationState.kt

```diff
@@ -121,6 +121,13 @@ private suspend fun CodeGenerationState.generateCode(codeGenerationId: String):
                 val newFileInfo = registerNewFiles(newFileContents = codeGenerationStreamResult.new_file_contents)
                 val deletedFileInfo = registerDeletedFiles(deletedFiles = codeGenerationStreamResult.deleted_files)
 
+                if (newFileInfo.isNotEmpty() || deletedFileInfo.isNotEmpty()) {
+                    messenger.sendAnswerPart(
+                        tabId = tabID,
+                        message =  message("amazonqFeatureDev.code_generation.generated_code")
```

Comment:
this should be different message on code re-generation

Reply:
Is the purpose of this PR include strings not currently handled?

Reply:
not sure if i understand, I thought updating customer facing texts are part of this PR since you have added this part where it adds a message upon code completion

Reply:
So, just giving more context to be clear: this string just updates an existing message that shows currently as "Generating code ..." even when finishes code, so the intent is provide a better updated string once its generated. 

What you're proposing is when we click "Retry" and goes to all code generation process again, passing through "Generating code ..." again, we should at the end have a different "messaging" which relates to "Here's your code (again)". Which is debatable, since I don't see the current messaging "Here's your code. You can choose a file to see a diff with the changes I'm proposing." needing to be different when its regenerated.

Reply:
Here is what is proposed for this string in the doc:

[FIRST GENERATION]
Here's your code. You can choose a file to see a diff with the changes I'm proposing.

[SECOND GENERATION]
I generated new code that incorporates your feedback. You can choose a file to see a diff with the changes I'm proposing.